ludwig/features/sequence_feature.py

```diff
@@ -121,6 +123,42 @@ def forward(self, v: TorchscriptPreprocessingInput):
         return sequence_matrix
 
 
+class _SequencePostprocessing(torch.nn.Module):
+    def __init__(self, metadata: Dict[str, Any]):
+        super().__init__()
+        self.max_sequence_length = int(metadata["max_sequence_length"])
+        self.idx2str = metadata["idx2str"]
+        self.unknown_symbol = UNKNOWN_SYMBOL
+        self.predictions_key = PREDICTIONS
+        self.probabilities_key = PROBABILITIES
+        self.probability_key = PROBABILITY
+
+    def forward(self, preds: Dict[str, torch.Tensor]) -> Dict[str, Any]:
+        """Takes a dictionary of tensors and returns a dictionary of tensors."""
+        pred_predictions = preds[self.predictions_key]
+        predictions: List[List[str]] = []
+        for sequence in pred_predictions:
+            sequence_predictions: List[str] = []
+            for i in range(self.max_sequence_length):
+                unit_id = int(sequence[i].item())
+                if unit_id < len(self.idx2str):
+                    unit_prediction = self.idx2str[unit_id]
+                else:
+                    unit_prediction = self.unknown_symbol
```

Comment:
When/Does the model actually predict `unit_id`s larger than `idx2str`?

IIRC, `idx2str` should already contain all of the special symbols, which includes the unknown symbol.